<launch>
    
    <arg name="merge_laser_frame" default="merge_laser_link"/>

    <group ns="robot1">
       <node pkg="laser_merge" type="laser_merge" name="laser_merge" output="screen">
        <remap from="cloud_out" to="/robot1/merge_laser/cloud"/>
        <param name="scan_topic_one" value="/robot1/kinect_scan"/>
        <param name="scan_topic_two" value="/robot1/urg_scan"/>
        <param name="merge_laser_frame" value="robot1/$(arg merge_laser_frame)"/>
    </node>

    <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan_node">
        <remap from="cloud_in" to="/robot1/merge_laser/cloud"/>
        <remap from="scan" to="merge_laser/scan"/>
        <!-- <param name="target_frame" value="$(arg merge_laser_frame)"/> -->
        <param name="transform_tolerance" value="0.01"/>
        <param name="min_height" value="-1.0"/>
        <param name="max_height" value="1.0"/>
        <param name="angle_min" value="-3.1416"/>
        <param name="angle_max" value="3.1416"/>
        <param name="range_min" value="0.0"/>
        <param name="range_max" value="10.0"/>
        
    </node> 
    </group>

    
</launch>